### PR TITLE
Fix tests for Swift 3.1

### PR DESCRIPTION
**Pull request by tali**

### CODE REVIEW

File: Sources/include/module.modulemap [hunk 1]
```diff
@@ -5,7 +5,7 @@ module HTMLKit {
 	export *
 
 	explicit module Private {
-		header "CSSCodePoints.h"
+		textual header "CSSCodePoints.h"
```
Comment (tali) [inline Sources/include/module.modulemap:8]: I'm not sure if this is the right way to fix it.
But it seams to works.

File: Tests/HTMLKitTests/CSSSelectorParserTests.m [hunk 2]
```diff
@@ -7,7 +7,7 @@
 //
 
 #import <XCTest/XCTest.h>
-#import <HTMLKit/HTMLKit.h>
```
Comment (tali) [inline Tests/HTMLKitTests/CSSSelectorParserTests.m:10]: Was `<HTMLKit/HTMLKit.h>` chosen on purpose?
I got 'file not found' here.

File: Sources/include/module.modulemap (hunk 1 above)
Comment (iabudiab) [reply Sources/include/module.modulemap:8]: Hey there! The textual header is treated analogous to the `#include` directive, i.e. the header is textually included and is the proper way for including a header multiple times. In this case headers with conflicting defines.

File: Tests/HTMLKitTests/CSSSelectorParserTests.m (hunk 2 above)
Comment (iabudiab) [reply Tests/HTMLKitTests/CSSSelectorParserTests.m:10]: There was no particular reason here and it can be replaced as far as I am concerned.

**Comment by iabudiab** [comment]
@tali Could you please merge against `develop` instead of `master`

**Comment by iabudiab** [comment]
Swift 3.1 tests are on my agenda for the coming weekend. Especially running the runtime tests, i.e. the `Tokenizer` and `Tree Construction` tests. These run fine when started in Xcode or via `xcodebuild` but are not detected using pure `swift test`.

The next thing that is bugging me is the umbrella and all the private headers. I have yet to find a clean solution to prevent all the `incomplete umbrella` warnings.

I'll be in touch tomorrow, since it is already 1:30 AM over here :)

**Comment by tali** [comment]
I didn't notice it at first, but the test suite is still failing when run in Swift 3.1:

```
Test Suite 'HTMLKitTokenizerPerformance' started at 2017-03-07 07:42:19.953
Test Case '-[HTMLKitTokenizerPerformance testTokenizerPerformance]' started.
2017-03-07 07:42:20.207 xctest[22261:6347726] NSScanner: nil string argument
zsh: exit 1     swift test
```

The call to `[NSString stringWithContentsOfFile:path encoding:NSUTF8StringEncoding error:nil]` tries to read from `.../Contents/Resources/Fixtures/HTML Standard.html` and returns `nil`.

**Comment by iabudiab** [comment]
@tali Hey there, I have taken a look at the whole testing thing with SwiftPM. The resources couldn't be loaded with SwiftPM because they are not copied into the testing bundle. In Xcode there is the "Copy Bundle Resources" phase that takes care of things.

I have introduced a small helper function to load the fixtures that should behave correctly for Xcode and SwiftPM testing. You can find it on `develop`.

Could you please make the PR against `develop` instead of `master` so I can merge it.

**Comment by tali** [comment]
Done, I created #8 for that.

**Pull request closed without merging**